src/pages/About/About.jsx

```diff
@@ -0,0 +1,57 @@
+import Image from "next/image";
+const About = () => {
+    return (
+        <div>
+            {/* Healing Part */}
+            <div class='flex flex-col h-[518]'>
+                <div>
+                    {/*  Healing 
+     (the subject is missing It`s component we will post it later) */}
+                </div>
+                <div className=' h-[294px] flex justify-center items-center'>
+                    {/*   Second div content goes here  */}
+                    <p className=' h-[234px] w-[1096px]  leading-[2rem] mt-[30px]'>
```

Comment:
Try as much as possible to give height to any space except in rare cases, as it is detrimental to responsiveness
In your case, you can use padding or margin to expand the space.
+ use (container) class for that idle-space in sides.

It's possible to be:
`<div className='container flex justify-center items-center py-20'>
                    <p className='mt-[30px]'> .... </p>
</div>`